verbose: true
play_file_flag: true
file_result_flag: false

joint_motors_sub_name: /motor_states/joints_port
joint_ctrl_pub_name: /hydrusx/joints_ctrl
joint_states_sub_name: /hydrusx/joint_states
uav_nav_pub_name: /uav/nav
uav_odom_sub_name: /uav/cog/odom
object_pos_sub_name: /object/2dpose

motion_func_loop_rate: 40.0
search_func_loop_rate: 0.0 # no joint pub from grasp form searching method
#nav_vel_limit: 0.3

approach_pos_threshold: 0.1
approach_yaw_threshold: 0.15
approach_count: 1.0

falling_speed: -0.1
ascending_speed: 0.1
grasping_height: 0.06
transportation_threshold: 0.15 # pos threshold to the box
transportation_count: 2.0
dropping_offset: 0.3

# red box
box_x: 0.540
box_y: 0.32
box_z: 0.3


rolling_motion_test: false
#func_loop_rate: 1.0
test_rolling_phi: 0.2